src/plugin/luapi_application.h

```diff
@@ -475,6 +476,194 @@ static int applib_changeBackgroundPdfPageNr(lua_State* L) {
     return 1;
 }
 
+/*
+ * Returns a table encoding all info on the pen in a Lua table of the shape
+ * {
+ *   "size" = string
+ *   "fill" = bool
```

Comment:
```suggestion
 *   "filled" = bool
```

I think `filled` makes more sense here; I associate `fill` with `fill details` (e.g. fill opacity, fill color, etc).